app/routes/user.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+const userController = require("../controllers/userController");
+const authMiddleware = require("../middleware/authMiddleware");
+
+router.get("/profile", authMiddleware, userController.getUserProfile);
```

Comment:
we discussed to remove  `/profile` as it doesn't make sense, right?

Reply:
So you mean need to just keep **/user** right?

Reply:
Updated.